src/components/RegisterForm/RegisterForm.tsx

```diff
@@ -0,0 +1,91 @@
+import { FormEvent, useState } from "react";
+import { InputField } from "../Input/Input";
+
+const RegisterForm = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [repeatPassword, setRepeatPassword] = useState("");
+  const [rememberMe, setRememberMe] = useState(false);
+  const [errors, setErrors] = useState({ email: "", password: "", repeatPassword: "" });
+
+  const handleSubmit = (e: FormEvent) => {
+    e.preventDefault();
+    let formIsValid = true;
+    const newErrors = { email: "", password: "", repeatPassword: "" };
+
+    if (!email) {

```

Comment:
Why do you make the form manually and not with the React Hook Form library?

Reply:
First I finished the design and operation then I put the react hook library in it